package/kedro_viz/api/rest/responses.py

```diff
@@ -267,3 +270,22 @@ def get_default_response() -> GraphAPIResponse:
         modular_pipelines=modular_pipelines_tree,
         selected_pipeline=default_selected_pipeline_id,
     )
+
+
+class EnhancedORJSONResponse(ORJSONResponse):
+    @staticmethod
+    def encode_to_human_readable(content: Any) -> bytes:
+        """A method to encode the given content to JSON, with the
+        proper formatting to write a human-readable file.
+
+        Returns:
+            A bytes object containing the JSON to write.
+
+        """
+        encoded_content = jsonable_encoder(content)
```

Comment:
Minor preference to put this outside this class. This would be consistent with how FastAPI handles the endpoint, where (as I understand it) it calls `jsonable_encoder` and _then_ the `Response`. Certainly there's no explicit call to `jsonable_encoder` in the `render` method of `JSONResponse` etc. anyway so it feels a bit off here.

Reply:
You're right, fixed in [4485b93](https://github.com/kedro-org/kedro-viz/pull/1087/commits/4485b9390dd8241e79f2da028aac5260745c8b69).